File: cpp/1.Patterns/5.inverted_right_pyramid.cpp
/*
Input Format: N = 3
Result: 
* * *
* * 
*
*/
#include<bits/stdc++.h>
using namespace std;
void print(int n)
{
   /*
   //1st way
    for(int i=n;i>0;i--)
        {
            for(int j=1;j<=i;j++)
            {
                cout<<"* ";
            }
            cout<<endl;
        }*/

    //2nd way
    for(int i=0;i<n;i++)
        {
            for(int j=n;j>i;j--)
            {
                cout<<"* ";
            }
            cout<<endl;
        }
   
}

int main()
{
    int n;
    cin>>n;
    print(n);
    return 0;
}

File: python/1.Patterns/5.inverted_right_pyramid.py
"""
 Input Format: N = 3
Result: 
* * *
* * 
* 
"""
def pattern5(n):
    for i in range(n): #0,n,1
        for j in range(n,i,-1):
            print("*",end=" ")
        print()
        pass
    
def alternative(n):
    for i in range(n,0,-1):
        for j in range(i):
            print("*",end=" ")
        print()
    

    
n=int(input())
pattern5(n)
print()
alternative(n)